RickAndMorty/Views/RMListView/Cells/RMCharacter/RMCharacterHorizontalCell.swift

```diff
@@ -92,8 +91,8 @@ final class RMCharacterHorizontalCell: UICollectionViewCell, RMItemCell {
         NSLayoutConstraint.activate([
             container.topAnchor.constraint(equalTo: self.topAnchor, constant: 0),
             container.bottomAnchor.constraint(equalTo: self.bottomAnchor, constant: 0),
-            container.leadingAnchor.constraint(equalTo: self.leadingAnchor, constant: 0),
-            container.trailingAnchor.constraint(equalTo: self.trailingAnchor, constant: 0),
+            container.leadingAnchor.constraint(equalTo: self.leadingAnchor, constant: 20),
```

Comment:
Can this be handled by the Compositional Layout ?

Reply:
Yes, it can